docs/configure-custom-values-scalardb-cluster.md

```diff
@@ -0,0 +1,230 @@
+# Configure a custom values file for ScalarDB Cluster
+
+This document explains how to create your custom values file for the ScalarDB Cluster chart. For more details on the parameters, see [README](https://github.com/scalar-labs/helm-charts/blob/main/charts/scalardb-cluster/README.md) of the ScalarDB Cluster chart.
+
+## Required configurations
+
+### Image configurations
+
+You must set `scalardbCluster.image.repository` and `scalardbCluster.image.tag`. Please specify the container repository information that you pull the ScalarDB Cluster container image.
+
+```yaml
+scalardbCluster:
+  image:
+    repository: <Container image of ScalarDB Cluster>
+    tag: <Tag of image>
+```
+
+### Database configurations
+
+You must set `scalardbCluster.scalardbClusterNodeProperties`. Please set your `scalardb-cluster-node.properties` to this parameter. For more details on the configurations of ScalarDB Cluster, see [ScalarDB Cluster document](https://github.com/scalar-labs/scalardb-cluster/blob/main/docs/index.md#configurations).
+
+```yaml
+scalardbCluster:
+  scalardbClusterNodeProperties: |
+    scalar.db.cluster.membership.type=KUBERNETES
+    scalar.db.cluster.membership.kubernetes.endpoint.namespace_name=${env:SCALAR_DB_CLUSTER_MEMBERSHIP_KUBERNETES_ENDPOINT_NAMESPACE_NAME}
+    scalar.db.cluster.membership.kubernetes.endpoint.name=${env:SCALAR_DB_CLUSTER_MEMBERSHIP_KUBERNETES_ENDPOINT_NAME}
+    scalar.db.contact_points=localhost
+    scalar.db.username=cassandra
+    scalar.db.password=cassandra
```

Comment:
We need to make these sensitive info environment variables for the security reason?

Reply:
Ah, that's right. It is better to use env variables in the example.

I set row values here since: 

* Strictly, using env variables is not a **requirement**.
* There is another section (`Secret configurations`) that describes how to use the `Secret` resource.

However, I think this is not a good example. Using env variables is strongly recommended in the production environment. So, I will update this example to use env variables.

Reply:
I did it in c6ada108210e988c5ca16487ad61c34d65e40792. Please take a look again when you have time!
(I will do the same update for other documents in another PR.)